src/huggingface_hub/hf_api.py

```diff
@@ -1857,6 +1861,10 @@ def create_commit(
             `str` or `None`:
                 If `create_pr` is `True`, returns the URL to the newly created Pull Request
                 on the Hub. Otherwise returns `None`.
+
+        Raises:
+            :class:`ValueError`:
+                If the Hub API returns an HTTP 400 error (bad request)
```

Comment:
Hey 👋 
I know it's not exactly related to this issue but I realized when reading the docs that the "Raises" part is never rendered by the autodoc (example: https://moon-ci-docs.huggingface.co/docs/huggingface_hub/pr_963/en/package_reference/hf_api#huggingface_hub.HfApi.create_commit). Is that a bug or a feature ? 🙄 

If a bug, I can create a separate issue on the repo to document it.

Reply:
cc @mishig25

Reply:
Seems to have been discussed here as well so I guess we are fine to keep it as it is for now.